_topic_maps/_topic_map.yml

```diff
@@ -585,10 +585,10 @@ Topics:
   File: understanding-upgrade-channels-release
 - Name: Understanding OpenShift update duration
   File: understanding-openshift-update-duration
-- Name: Preparing to update to OpenShift Container Platform 4.12
+- Name: Preparing to update to OpenShift Container Platform 4.13
```

Comment:
Can you back out the changes to this file? I have another PR that will take care of everything needed in this file (#55224)